src/main/kotlin/racing/InputView.kt

```diff
@@ -5,6 +5,29 @@ class InputView {
         print(prompt)
         return readln().toPositiveIntOrNull() ?: retryGet(prompt)
     }
+    
+    fun getNames(prompt: String): List<Car> {
+        print(prompt)
+        val cars = readln().toCars()
+        return cars.ifEmpty {
+            retryGetNames(prompt)
+        }
+    }
+
+    private fun retryGetNames(prompt: String): List<Car> {
+        println("다시 입력해 주세요.")
+        return getNames(prompt)
+    }
+    
+    private fun String.toCars(): List<Car> {
+        val names = this.split(",")
+        val cars = mutableListOf<Car>()
+        repeat(names.size) {
+            cars.add(Car(names[it]))
+        }
+        
+        return cars
```

Comment:
```suggestion
        return this.split(",")
            .map { Car(it) }
```
stream 문법을 활용하는 방법도 있습니다 :)

Reply:
@jinyoungchoi95 
[d988e56](https://github.com/next-step/kotlin-racingcar/pull/1553/commits/d988e562f8e1747b2c68cdd9d76df6da026b3e75) 수정했습니다.